### ISSUE TITLE
Accessibility - button colour does not pass colour contrast checks

### ISSUE THREAD

**Issue by chris-gds** [ant-design/ant-design#45144]
### Reproduction link

[https://ant.design/components/button](https://ant.design/components/button)

### Steps to reproduce

Hey peeps, offering some free accessibility feedback:

1. Visit the button.
2. Check colour contrast.

### What is expected?

I would expect this to pass AA. 
Review: https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

### What is actually happening?

It's failing

![screenshot of AntDesign button components failing colour contrast](https://github.com/ant-design/ant-design/assets/71266765/62545b92-6652-454f-b44b-77990d1adbbd)

| Environment | Info |
| --- | --- |
| antd | 5.9.4 |
| React | All |
| System | All |
| Browser | All |

---

###  Solution:
A simple fix is to darken the blue, if #0E6EF6 is used instead this passes

**Comment by MadCcc** [comment]
It's all by design, but you can custom brand color by ConfigProvider.

**Comment by chris-gds** [comment]
I've removed the WCAG reference as I think that's misleading. However is more information: 
https://axesslab.com/disabled-buttons-suck/

**Comment by chris-gds** [comment]
@MadCcc that is unfortunate that you would rather prioritise aesthetics rather than technical standards. 
Can I ask which people are you willing to exclude from this design?
